common/thrift_router.cpp

```diff
@@ -51,6 +51,11 @@ bool parseHost(const std::string& str, common::detail::Host* host,
     return false;
   }
   auto group = (tokens.size() == 3 ? tokens[2] : "");
+  // Based on format of generated shard config which is
+  //    ${ip_address}:${thrift_service_port}:${az}_${pg}
+  // as specified in ConfigGenerator.java, we try to cache the az information.
```

Comment:
Nit, but I think it makes it more clear to move this section of comments above line 53. Or above 43, to be even more clear.

Reply:
done

Reply:
done